README.md

```diff
@@ -18,8 +18,16 @@
   * Alternatively run `./node_modules/.bin/ava test/unit/advanced-camera-test.js` for an individual test
 * `npm run coverage` runs the code coverage report
   * `npm test` will run both lint and all the unit-test and report coverage in text form in command line
+* `npm start` will run webpack with the `--watch` flag and serve the test-page using stupid server. 
```

Comment:
I added this (and the `concurrently` & `stupid-server` dev dependencies because stopping, rebuilding, and then serving the test-page over and over got really old. I would be open to suggestions if there are better ways and/or tools for doing this.